### PR TITLE
feat: Creating an example of an email summary

### PR DESCRIPTION
**Pull request by TiagoVictor**

### CODE REVIEW

File: essentials/function.mdx [hunk 1]
```diff
@@ -325,6 +325,119 @@ print(summarizedResponse.json()['choices'][0]['message']['content'])
...
+        mail.logout()
+        return json.dumps(email_information)
+    except Exception as e:
+        print("Err:", str(e))
+```
+So, we send the emails to a new function that will summarize
+
+```python
+summarizedResponse = {
+  "model": "codellama-7b-instruct",
```
Comment (TeodoroRaulino) [inline essentials/function.mdx]: Why are you using two different models?

**Review by TeodoroRaulino — APPROVED** [review]

File: essentials/function.mdx [hunk 2]
```diff
@@ -325,6 +325,119 @@ print(summarizedResponse.json()['choices'][0]['message']['content'])
 ```
 It´s currently moderate rain in London with a temperature of 8.0 degrees Celsius.
 ```
+### Example 4: `get_email_resume`
```
Comment (TeodoroRaulino) [inline essentials/function.mdx]: @fuchsfelipel , what do you think of the idea of ​​function calling having just one example that is already very complete and adding the other examples to a page that only contains examples ? So each example can be better detailed on this examples page

Comment (TeodoroRaulino) [reply essentials/function.mdx]: Could be on another task

File: essentials/function.mdx (hunk 1 above)
Comment (TiagoVictor) [reply essentials/function.mdx]: Corrected to use only one model

File: essentials/function.mdx (hunk 2 above)
Comment (fuchsfelipel) [reply essentials/function.mdx]: I agree! In fact, for this specific example, I'd move it to a page called `Tutorial` and give a step-by-step explanation

File: essentials/function.mdx [hunk 3]
```diff
@@ -325,6 +325,132 @@ print(summarizedResponse.json()['choices'][0]['message']['content'])
...
+For this example we will use gmail as an email service
+- Objective: Create a resume from your e-mails
+- Parameter: `value` (desired quantity of e-mails), `login` (your e-mail), `password` (your e-mail password)
+- Usage Example: User provides a location and receive local weather information
+
```
Comment (fuchsfelipel) [inline essentials/function.mdx]: When you move this to the `Tutorial` page, give an explanation of how it works:

Eg. On the code below, what actually happens is:
1. We send the request to the LLM
2. The LLM sees your emails using IMAP
3. ....

File: essentials/function.mdx [hunk 4]
```diff
@@ -325,6 +325,132 @@ print(summarizedResponse.json()['choices'][0]['message']['content'])
...
+### Request
+Firstly, you need to create a password for less secure apps by following the link: https://support.google.com/a/answer/6260879?hl=en.
+With the password created, we can make the first request to extract the email, password and number of emails that will be summarized.
+```python
+api_request_json = {
+  "model": "llama-70b-chat",
+  "messages": [
+    {"role": "user", "content": "Make a resume from my last e-mail, login: tiagovictordeoliveira@gmail.com and password: nnmy fvbx sucz gxdu"},
```
Comment (fuchsfelipel) [inline essentials/function.mdx]: This is a big security problem.... your email and password are now public to the world! I imagine you already expired your password... but change your email so you don't get too much SPAM messages

Comment (fuchsfelipel) [inline essentials/function.mdx]:
```suggestion
    {"role": "user", "content": "Make a summary of my last e-mail, login: tiagovictordeoliveira@gmail.com and password: nnmy fvbx sucz gxdu"},
```

File: essentials/function.mdx [hunk 5]
```diff
@@ -325,6 +325,132 @@ print(summarizedResponse.json()['choices'][0]['message']['content'])
...
+  ],
+  "functions": [
+    {
+        "name": "get_email_resume",
```
Comment (fuchsfelipel) [inline essentials/function.mdx]: Rename to `get_email_summary`

In english:
`resume` quer dizer `continuar``
`summary` quer dizer `resumo`

**Review by fuchsfelipel — CHANGES_REQUESTED** [review]

File: essentials/function.mdx (hunk 4 above)
Comment (fuchsfelipel) [reply essentials/function.mdx]: https://stackoverflow.com/questions/2110595/redaction-in-git

**Pull request closed without merging**